tendrl/node_agent/tests/conftest.py

```diff
@@ -0,0 +1,42 @@
+import os
+import pytest
+import re
+
+
+@pytest.fixture
+def definitions_path():
+    """Get absolute path to definitions file."""
+
+    return os.path.abspath(os.path.dirname(os.path.realpath(__file__)) +
+                           "/../../../etc/tendrl_definitions_node_agent.yaml")
+
+
+def get_defined_functions(file_path):
+    """Get list of runable functions from definitions."""
+
+    functions = []
+    list_search = False
+    with open(file_path, 'r') as f:
+        for line in f:
+            if not list_search:
```

Comment:
I think this `if` can be outside of `for` cycle. Because now it is checked for every line and I think that is not efficient.